superbench/tools/system_info.py

```diff
@@ -0,0 +1,447 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+"""Generate system config."""
+
+import json
+import subprocess
+import xmltodict
+from pathlib import Path
+
+
+class SystemInfo():
+    """Systsem info class."""
+    def __run_cmd(self, command, non_root=False):
+        """Run the command as root or non-root user and return the stdout string..
+
+        Args:
+            command (string): the command to run in terminal.
+            non_root (bool): the mode to run the command, root or non-root.
+
+        Returns:
+            string: the stdout string of the command.
+        """
+        prefix = ''
+        if not non_root:
+            prefix = 'sudo '
```

Comment:
what if `sudo` is not installed? and usually there's password for `sudo`

Reply:
seems non_root is not used, can you remove `sudo` here and let others run `sudo python system_info.py` directly?

Reply:
ok